packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -321,36 +321,21 @@ class SlackBridge {
 
 	/**
 	 * We have received a message from slack and we need to save/delete/update it into rocket
+	 * https://api.slack.com/events/message
 	 */
```

Comment:
There is too much going on this method.  Going to start slowly refactoring these out into single methods